### ISSUE TITLE
Periodically recieving "missing revert data in call exception" on gateway.

### ISSUE THREAD

**Issue by Kally95** [omgnetwork/optimism#602]
![error](https://user-images.githubusercontent.com/40044444/144250634-be3d0f62-745f-4643-9f12-9fa3a3e77008.png)

Having imported an NFT contract via the NFT section on Boba Gate Way, I periodically receive the following error (around <30 seconds):

"missing revert data in call exception (error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32000,\"message\":\"execution reverted\"},\"id\":370}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x470a5532d95c45108f86dd2990ef98f406dec432\",\"data\":\"0x70a08231000000000000000000000000b048f1d92ace4542e8a1df8a624463e2162498d7\"},\"latest\"],\"id\":370,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://mainnet.boba.network"}, data="0x", code=CALL_EXCEPTION, version=providers/5.4.5)"

It's possible I imported the wrong address, however, the error message is very persistent. It shows up on all pages and every 20-30 seconds if that. 

****Temp fix:**** as pointed out by @claude-mon you can reset your browser settings although this clears any existing contracts you had imported so you'll need to reimport (i.e your boba punks). Would it be possible to introduce a way to delete/update imported contract addresses, so we can manage the contracts we've imported?

**Comment by CAPtheorem** [comment]
Yes, good suggestion - that functionality will be rolled out by Monday. Briefly, you will see a list of all the contract you have added and you will be able to remove contracts (and associated NFTs).
